### DISCUSSION TITLE
Is there a way to add a 'kid' to the header?

### DISCUSSION THREAD

**Discussion by DucatiScotty** [DigitalAXPP/jwtPS#36] [Q&A] [answered]
I'm trying to generate a JWT for the App Store API.  

My current header looks like this:

> {
>   "typ": "JWT",
>   "alg": "ES256"
> }

but I need it to look like this:

> {
>   "typ": "JWT",
>   "kid": "WE35634",
>   "alg": "ES256"
> } 

Is there a way to include that additional information in the header?

Thank you,
Scott

**Comment by DigitalAXPP** [comment]
Hi Scott,


At the moment, this option is not available but it is on my mental list.

I am however not sure when I will be able to include it as it doesn't appear to be straight forward when I first attempted it.

 

Best regards,

Alex

 

Gesendet: Donnerstag, 31. August 2023 um 20:37 Uhr
Von: "DucatiScotty" ***@***.***>
An: "DigitalAXPP/jwtPS" ***@***.***>
Cc: "Subscribed" ***@***.***>
Betreff: [DigitalAXPP/jwtPS] Is there a way to add a 'kid' to the header? (Discussion #36)


 

I'm trying to generate a JWT for the App Store API.

My current header looks like this:


{
"typ": "JWT",
"alg": "ES256"
}


but I need it to look like this:


{
"typ": "JWT",
"kid": "WE35634",
"alg": "ES256"
}


Is there a way to include that additional information in the header?

Thank you,
Scott

**Comment by DucatiScotty** [comment]
Well that's a bummer.  Thanks for the quick reply.

**Reply by DigitalAXPP** [reply, marked as answer]
Hi Scott,
in the latest version it is now possible to provide your own header table.
